src/api/search/src/routes/health.js

```diff
@@ -0,0 +1,11 @@
+const { Router } = require('@senecacdot/satellite');
```

Comment:
You can drop this, and use https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/16.x/api-cluster-health.html in the `healthCheck` for the `Satellite` constructor.